lhotse/bin/modes/recipes/eval2000.py

```diff
@@ -6,16 +6,28 @@
 from lhotse.recipes import prepare_eval2000
 from lhotse.utils import Pathlike
 
+EVAL2000_TRANSCRIPT_DIR = "LDC2002T43"
+
 
 @prepare.command(context_settings=dict(show_default=True))
 @click.argument("corpus-dir", type=click.Path(exists=True, file_okay=False))
 @click.argument("output-dir", type=click.Path())
+@click.argument(
```

Comment:
Have you tested this with and without providing `transcript-dir`? It seems you are expecting this to be a positional argument which may not be what we want.

Reply:
You are right, I'll look into it.